cmd/zed/permission.go

```diff
@@ -24,6 +32,8 @@ func registerPermissionCmd(rootCmd *cobra.Command) {
 	permissionCmd.AddCommand(checkCmd)
 	checkCmd.Flags().Bool("json", false, "output as JSON")
 	checkCmd.Flags().String("revision", "", "optional revision at which to check")
+	checkCmd.Flags().Bool("trace", false, "requests the debug information of the check from SpiceDB and prints out a trace of the requests")
```

Comment:
I'm not convinced that these flag names make sense. `trace` is so overloaded with distributed tracing and logging.

Maybe something like:
`--resolution-path`
`--resolution-schema`

Reply:
Those are verbose to type though; I was going for a short, simple name. Maybe... `--debug`?